octue/resources/service.py

```diff
@@ -0,0 +1,206 @@
+import json
+import logging
+import uuid
+from concurrent.futures import TimeoutError
+import google.api_core.exceptions
+from google.api_core import retry
+from google.cloud import pubsub_v1
+
+from octue import exceptions
+from octue.mixins import CoolNameable
+
+
+logger = logging.getLogger(__name__)
+
+
+OCTUE_NAMESPACE = "octue.services"
+ANSWERS_NAMESPACE = "answers"
```

Comment:
Is this the namespace on pubsub? if so make it octue.answers

Reply:
It's a sub-namespace of a service in `octue.services` e.g. 
```
octue.services.6a05b695-4aa1-468c-bdf5-73303665165d.answers.147920905457550709731291009228875486831
```